packages/hydrogen/src/product/useVariantUrl.example.tsx

```diff
@@ -0,0 +1,19 @@
+import {Link} from '@remix-run/react';
+import {useVariantUrl} from '@shopify/hydrogen';
+
+function MyComponent() {
+  const {to, search} = useVariantUrl('snowboard', [
```

Comment:
We can't return a real `URL` object, because we don't easily have the origin at the moment, because `useLocation()` doesn't return it. I felt like `path` or `pathname` implies it doesn't include the `search` (because the native `URL.prototype.pathname` doesn't).